.evergreen/run-fuzz.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o errexit  # Exit the script with error if any of the commands fail
+
+FUZZTIME=10m
+
+# Change the working directory to the root of the mongo repository directory
+cd $PROJECT_DIRECTORY
+
+# Get all go test files that contain a fuzz test.
+FILES=$(grep -r --include='**_test.go' --files-with-matches 'func Fuzz' .)
+
+# For each file, run all of the fuzz tests in sequence, each for -fuzztime=FUZZTIME.
+for FILE in ${FILES}
+do
+	PARENTDIR="$(dirname -- "$FILE")"
+
+	# Clean testing cache.
+	go clean -testcache
+	go clean -fuzzcache
+
+	# Get a list of all fuzz tests in the file.
+	FUNCS=$(grep -o 'func Fuzz[A-Za-z0-9]*' $FILE | cut -d' ' -f2)
+
+	# For each fuzz test in the file, run it for FUZZTIME.
+	for FUNC in ${FUNCS}
+	do
+		echo "Fuzzing \"${FUNC}\" in \"${FILE}\""
+
+		# Create a set of directories that are already in the subdirectories testdata/fuzz/$fuzzer corpus. This
+		# set will be used to differentiate between new and old corpus files.
+		declare -a cset
+
+		if [ -d $PARENTDIR/testdata/fuzz/$FUNC ]; then
+			# Iterate over the files in the corpus directory and add them to the set.
+			for SEED in $PARENTDIR/testdata/fuzz/$FUNC/*
+			do
+				cset+=("$SEED")
+			done
+		fi
+
+		go test ${PARENTDIR} -run=${FUNC} -fuzz=${FUNC} -fuzztime=${FUZZTIME} || true
+
+		# Check if any new corpus files were generated for the fuzzer. If there are new corpus files, move them
+		# to $PROJECT_DIRECTORY/fuzz/$FUNC/* so they can be tarred up and uploaded to S3.
+		if [ -d $PARENTDIR/testdata/fuzz/$FUNC ]; then
+			# Iterate over the files in the corpus directory and check if they are in the set.
+			for CORPUS_FILE in $PARENTDIR/testdata/fuzz/$FUNC/*
+			do
+
+				# Check to see if the value for CORPUS_FILE is in cset.
+				if [[ ! " ${cset[@]} " =~ " ${CORPUS_FILE} " ]]; then
+					# Create the directory if it doesn't exist.
+					if [ ! -d $PROJECT_DIRECTORY/fuzz/$FUNC ]; then
+						mkdir -p $PROJECT_DIRECTORY/fuzz/$FUNC
+					fi
+
+					# Move the file to the directory.
+					mv $CORPUS_FILE $PROJECT_DIRECTORY/fuzz/$FUNC
+
+					echo "Moved $CORPUS_FILE to $PROJECT_DIRECTORY/fuzz/$FUNC"
```

Comment:
```suggestion
					echo "Moved new $CORPUS_FILE to $PROJECT_DIRECTORY/fuzz/$FUNC"
```

Reply:
This is just a log notifying the user that generated data has been moved to the file which will be tarred and put on S3. It will read like 

```
Moved da39a3ee5e6b4b0d3255bfef95601890afd80709 to /fuzz/FuzzDecode
```

In this case "new" seems redundant.

Reply:
Fair, sounds good.